File: problemas/1100/1158.js
const entrada = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');
const n = parseInt(entrada.shift());

for (let i = 0; i < n; i++) {
  let [x, y] = entrada.shift().split(' ').map(z => Number(z));
  let soma = 0;

  if (x % 2 === 0) {
    x++;
  }

  for (let j = 0; j < y * 2; j += 2) {
    soma += x + j;
  }

  console.log(soma);
}

File: problemas/1100/1158.dart
import 'dart:io';

void main() {
  final n = int.parse(stdin.readLineSync());

  for (var i = 0; i < n; i++) {
    final entrada = stdin.readLineSync().split(' ');
    int x = int.parse(entrada[0]);
    final y = int.parse(entrada[1]);
    int soma = 0;

    if (x % 2 == 0) {
      x++;
    }

    for (int j = 0; j < y * 2; j += 2) {
      soma += x + j;
    }

    stdout.writeln(soma);
  }
}